bcgov/HOUS-permit-portal
# [bphh-1020] Add validation to check for valid html for hint in requirements and display description in requirement block

## Description
[bphh-1020] Add validation to check for valid html for hint in requirements and display description in requirement block


## What type of PR is this? (check all applicable)

- [ x] 🐛 Bug Fix

## Related Tickets & Documents
https://hous-bssb.atlassian.net/browse/BPHH-1020


## Steps to QA
- Add invalid html to any hint in the Core Requirement excel file
- Run seeds
- The seeds should print out a validation error for the requirement
